src/effects/effectprocessor.h

```diff
@@ -32,7 +35,7 @@ class EffectProcessor {
     // samples. The group provided allows the effect to maintain state on a
     // per-group basis. This is important because one Effect instance may be
     // used to process the audio of multiple channels.
-    virtual void process(const QString& group,
+    virtual void process(const ChannelHandle& group,
```

Comment:
I think I do want to change the variable name for ChannelHandle since we're still going to see QString group all over the place, and it will get confusing.  

proposals:  handle, group_handle, hGroup, group_h, h_group, ... ch_handle, hChannel... something like that to make it distinct.  Yeah I know that's a lot of refactoring